### ISSUE TITLE
support python3

### ISSUE THREAD

**Issue by mbjones** [DataONEorg/slinky#1]
Current processing code depends on python 2.x. Need to upgrade to support 3.x.

**Comment by amoeba** [comment]
I've done this locally and was just waiting on us to pick a repo name to push it up. The migration seemed to go off without any hitches but I'm expecting we'll find some issue with common things such as string handling that the tests aren't catching already.

**Comment by mbjones** [comment]
There is a branch `feature_1_python3` with your code in it I think, not yet merged into develop.

**Comment by amoeba** [comment]
Ah, thanks for the heads up @mbjones. @ThomasThelen is currently kicking the tires with the service locally and will merge to develop if things work well for him.

**Comment by ThomasThelen** [comment]
I'm beginning work on this today, starting by rebasing it onto the `feature_kubernetes_deployment` branch in anticipation of it being merged into `develop` soon. Then, I'll work on testing the K8 deployment to make sure the results are the same between branches.
